pom.xml

```diff
@@ -362,6 +362,12 @@
         <artifactId>jclouds-slf4j</artifactId>
         <version>${version.jclouds}</version>
       </dependency>
+      
+      <dependency>
+        <groupId>org.jboss.resteasy</groupId>
+        <artifactId>resteasy-jaxrs</artifactId>
```

Comment:
Ok, understood. Using a JAX-RS 2.0 Client is a perfect use case when you need an implementation. Thanks for clarification.